src/tools/rbd_ggate/main.cc

```diff
@@ -17,7 +17,7 @@
 #include <boost/algorithm/string/predicate.hpp>
 #include <regex>
 
-#include "common/Formatter.h"
+#include "common/JSONFormatter.h"
```

Comment:
`src/tools/rbd_ggate/main.cc` uses `XMLFormatter` too -- it needs to be included just like in `src/tools/rbd_nbd/rbd-nbd.cc`.